lib/ansible/modules/clustering/consul.py

```diff
@@ -192,6 +200,13 @@
     interval: 60s
     http: http://localhost:80/status
 
+- name: register nginx with a tcp check
+  consul:
+    service_name: nginx
```

Comment:
nginx is something you'd most likely check with the http check, might be worth changing this to a non-http service to illustrate the difference.